autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -637,6 +637,22 @@ def predict_proba_model(self, X, model, level_start=0):
         if isinstance(model, str):
             model = self.load_model(model)
         X = self.get_inputs_to_model(model=model, X=X, level_start=level_start, fit=False)
+        EPS = 1e-8 # predicted probabilities can be at most this confident if we normalize predicted probabilities
```

Comment:
ditto on comment regarding normalization

Reply:
I will decrease the `EPS` factors so you can be assured they won't have any detrimental impact on performance ever.

Reply:
ditto on follow-up comment, add TODO to move normalization internal to models with Trainer dictating which models are normalized based on model problem_type